### ISSUE TITLE
Training difficulties

### ISSUE THREAD

**Issue by cbasavaraj** [ZhaoJ9014/face.evoLVe.PyTorch#52]
Hi,

I am training on casia-clean + align, using IR_SE_50 backbone and ArcFace head. Somehow the network just isn't learning well. The loss is around 24 for the first few epochs, and even after stage 1 adjustment, it drops only to around 19. 
I see during your training that the loss starts around 10. Anything I am doing wrong / missing?

Thanks

**Comment by ZhaoJ9014** [comment]
> Hi,
> 
> I am training on casia-clean + align, using IR_SE_50 backbone and ArcFace head. Somehow the network just isn't learning well. The loss is around 24 for the first few epochs, and even after stage 1 adjustment, it drops only to around 19.
> I see during your training that the loss starts around 10. Anything I am doing wrong / missing?
> 
> Thanks

Hi, thank you for your question. What you have done is correct. As you can see, the screenprint of training process in this repo is already after 10 epoches. The loss value of margin based function  is usually larger than vanilla softmax.

**Comment by closerbibi** [comment]
Hi @ZhaoJ9014, 
do you mean the figure here: 
![image](https://user-images.githubusercontent.com/8993498/79315346-59836900-7f35-11ea-8cdc-3576480e1ce0.png)
 is logged after training 10 epochs? Then what's do you do for the first 10 epochs? 

Thanks
